### PR TITLE
For #390: Fix broken contributing guide link in readme.md

### PR DESCRIPTION
**Pull request by bertya**
For #390: Fix broken contributing guide link in readme.md.
Though mentioned in the issue we probably gonna move the guide, I'm still pointing to the current location in the readme.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ npm install pcln-design-system
 If you'd like to contribute to the design system, we'd love to have your help.
 If you work at Priceline.com, please join the #design-system Slack channel and let us know what you'd like to work on.
 
-For all contributors, please be sure to read the [Contributing](docs/Contributing.md) doc.
+For all contributors, please be sure to read the [Contributing](docs/pages/Contributing.md) doc.
```
Comment (BeniCheni) [inline README.md]: @bertya, thanks for your 1st contribution!

Seems that [a suggestion](https://github.com/pricelinelabs/design-system/issues/390#issuecomment-427847467) in #390 is advising to moving the `Contributing.md` to the root. Would you mind updating?

**Comment by bertya** [comment]
@BeniCheni Updated.

**Comment by jxnblk** [comment]
Looks great, thanks! It'd be good to update the link from the docs site as well – should be this one line https://github.com/pricelinelabs/design-system/blob/master/docs/src/navigation.js#L56

**Comment by bertya** [comment]
@jxnblk can you help me on how should I update the link? Should I put it this way "https://github.com/pricelinelabs/design-system/blob/master/CONTIRBUTING.md"? but it looks weird to me.

**Comment by jxnblk** [comment]
@bertya Yes, that should be the canonical URL for that file on the master branch (assuming that it's spelled `CONTRIBUTING.md`)

**Comment by jxnblk** [comment]
LGTM :shipit:

**Pull request merged**